[ADD] purchase: RFQ (#10388)
odoo/documentation: content/applications/inventory_and_mrp/purchase/manage_deals/rfq.rst

~~~~~~~~~~~~~~~~~

   :guilabel:`Vendor reference` points to the sales and delivery order numbers sent by the vendor.
   This will come in handy once products are received and the PO needs to be matched to the delivery
   order.

   Under :guilabel:`Agreement`, blanket orders are long-term purchase agreements on recurring orders
   with set pricing. To view and configure blanket orders, head to
   :menuselection:`Purchase app --> Orders --> Purchase agreements`.

   :guilabel:`Currency` can be changed if purchasing products from a vendor in another country.
   To change the main currency or enable additional currencies, head to
   :menuselection:`Configuration --> Settings --> Currencies`.

   :guilabel:`Deliver to` provides options for the order shipment. Select the receiving warehouse
   address here, or select :guilabel:`Dropship` to indicate that this order is to be shipped
   directly to the end customer. When :guilabel:`Dropship` is selected, the
   :guilabel:`Dropship address` field is enabled. Contact names will populate here from the
   Contacts app.

.. tip::
   Expected Arrival is automatically calculated based on the Order Deadline and Vendor Lead Time.

>    Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at delivery.

Reviewer: Content in these admonition blocks are supplementary information. Since the expected arrival field is very important, I'd take it out and have it join its friends in the paragraph above.

In the products tab, add the products to be ordered. Select :guilabel:`Add a product` and start
typing in the product name to select the item from the drop-down menu. :guilabel:`Catalog` can also
be selected to navigate to a products menu under the chosen vendor. From here, products can be
added to the cart. If ordering a product that has not been configured, type in the product name,
select :guilabel:`Product Name` and manually add the unit price. Clicking into the product name
navigates to its product page, where purchasing details for future orders can be configured.

Send RFQ
--------

There are a few options to send an RFQ to vendors:

:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email

> address (configured in Contacts). The subject and body can be directly edited from Odoo and the RFQ
> pdf can be edited before sending. Once the email has been sent, the order moves into the

Reviewer: While it's definitely cool that we can send emails through Odoo, this doc is about the process of creating RFQs and |POs|. To keep the focus on that, I'm making a suggestion to gloss over these emailing features.

```suggestion
address (configured in the **Contacts** app). After crafting the desired message, click :guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the
```

> **RFQ Sent** stage.

Reviewer: Since it's text that we see explicitly on the UI
```suggestion
:guilabel:`RFQ Sent` stage.
```

> :guilabel:`Print RFQ` pulls the order information into a printable document that can be handed to
> the vendor or sent by mail.

Reviewer: ```suggestion
:guilabel:`Print RFQ` downloads a PDF version of the |RFQ|.
```

Confirm order
-------------

> Clicking :guilabel:`Confirm Order` directly transforms the RFQ into an active order. This can be
> selected once the RFQ has been sent and a confirmation from the vendor has been received, or
> directly from the RFQ stage if there is a prior agreement.

Reviewer: Suggestion to remove since there are basically no conditions needed to fulfill to click the Confirm Order button. (Except for entering a Vendor and Order Deadline, but you need that for an RFQ too)
```suggestion
Clicking :guilabel:`Confirm Order` directly transforms the RFQ into an active order.
```

.. tip::
   Odoo tracks communications on each order through the chatter screen on the right. This shows
   the emails sent between the user and the contact, as well as any internal notes and activities.
   Messages, notes, and activities can also be logged on chatter.

> Once an RFQ is confirmed, it creates a Purchase Order (PO). Products have been ordered!

Reviewer: We can also elaborate that the Order Deadline field on an RFQ has turned into the Confirmation Date, displaying the date and time the user confirmed the order.

.. note::

>    Odoo’s smart buttons facilitate the flow of orders. When the order is in the Purchase Order
>    stage, the next step is :guilabel:`Receive Products` which is selected as a confirmation once
>    the order has been delivered.

Reviewer: Nitpick: the Receive Products button is a normal button, and not a smart button. (The smart button is Receipts in the image below)
```suggestion
   After an order is placed, clicking :guilabel:`Receive Products` records the reception of new products into the database.
```

.. note::
   With the Inventory app installed, confirming a PO automatically creates a receipt document,

>    with the product information and expected arrival dates automatically populated.

Reviewer: we can also put a seealso admonition block at the end to bring attention to the next stage after receiving products, which is paying the vendors for the products they supplied us

```suggestion
   with the product information and expected arrival dates automatically populated.
   
.. seealso:
   (For Dina to do: link to this doc using the "doc" directive: https://www.odoo.com/documentation/16.0/applications/inventory_and_mrp/purchase/manage_deals/manage.html)
```

[RST cheatsheet for linking to other pieces of documentation](https://www.odoo.com/documentation/16.0/contributing/documentation/rst_cheat_sheet.html#documentation-pages-hyperlinks)

Reviewer: Great job on this, Dina!! :heart_eyes:

select :guilabel:`Product Name` and manually add the unit price.

.. note::
  To make adjustments to products and prices, access the product form by clicking the
  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
  the product name.

Send |RFQ|
----------

There are a few options on how to send an |RFQ| to vendors:

:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email
address (configured in the **Contacts** app). After crafting the desired message, click
:guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the :guilabel:`RFQ Sent` stage.

:guilabel:`Print RFQ` downloads a PDF version of the |RFQ|.

Confirm order
-------------

Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active order.

.. tip::
   Odoo tracks communications on each order through the chatter screen on the right. This shows
   the emails sent between the user and the contact, as well as any internal notes and activities.
   Messages, notes, and activities can also be logged on chatter.

Once an |RFQ| is confirmed, it creates a |PO|. Products have been ordered!

> On the new |PO|, the :guilabel:`Order Deadline` turns into the :guilabel:`Confirmation Date`, which

Reviewer: ```suggestion
On the new |PO|, the :guilabel:`Order Deadline` turns into :guilabel:`Confirmation Date`, which
```

displays the date and time the user confirmed the order.

> Depending on the user's chosen configuration in the *Purchase* app settings, a *vendor bill* is

Reviewer: ```suggestion
Depending on the user's chosen configuration in the **Purchase** app settings, a *vendor bill* is
```

created once products have been ordered or received. For more information on managing vendor bills,

> refer to :doc:`this documentation.<manage>`

Reviewer: Minor nitpick-- we typically have a space between link text ("this documentation"), and the link itself ("manage")
```suggestion
refer to :doc:`this documentation <manage>`.
```

.. note::
   After an order is placed, clicking :guilabel:`Receive Products` records the reception of new
   products into the database.

.. note::

>    With the *Inventory* app installed, confirming a *PO* automatically creates a receipt document,

Reviewer: We bold app names now (as of a couple days ago :sweat_smile: )
```suggestion
   With the **Inventory** app installed, confirming a *PO* automatically creates a receipt document,
```

typing in the product name to select the item from the drop-down menu. :guilabel:`Catalog` can also
be selected to navigate to a products menu under the chosen vendor. From here, products can be
added to the cart. To create a new product and add it, begin by typing the product name, select the
:guilabel:`Create [product name]` and manually add the unit price.

.. note::
  To make adjustments to products and prices, access the product form by clicking the
  :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
  the product name.

Send |RFQ|
----------

There are a few options on how to send an |RFQ| to vendors:

:guilabel:`Send by Email` automatically loads an RFQ template email to send to the vendor’s email
address (configured in the **Contacts** app). After crafting the desired message, click
:guilabel:`Send`. Once the email has been sent, the |RFQ| moves to the :guilabel:`RFQ Sent` stage.

:guilabel:`Print RFQ` downloads a PDF version of the |RFQ|.

Confirm order
-------------

Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active order.

.. tip::

>    Odoo tracks communications on each order through the chatter screen on the right. This shows
>    the emails sent between the user and the contact, as well as any internal notes and activities.
>    Messages, notes, and activities can also be logged on chatter.

Reviewer 2: ```suggestion
   Odoo tracks communications on each order through the chatter of the |PO| form. This shows the
   emails sent between the user and the contact, as well as any internal notes and activities.
   Messages, notes, and activities can also be logged on chatter.
```
depending on the zoom/browser size, the chatter can appear on the right *or* the bottom, so it's always best to keep that kind of tip as generic as possible.

> Once an |RFQ| is confirmed, it creates a |PO|. Products have been ordered!

Reviewer 2: ```suggestion
Once an |RFQ| is confirmed, it creates a |PO|.
```
While the enthusiasm is **always** appreciated 😄 we've been instructed to avoid using any exclamation points. Just gotta stick with boring periods, I'm afraid 😆

> On the new |PO|, the :guilabel:`Order Deadline` turns into :guilabel:`Confirmation Date`, which

Reviewer 2: ```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes :guilabel:`Confirmation Date`, which
```

displays the date and time the user confirmed the order.

Depending on the user's chosen configuration in the **Purchase** app settings, a *vendor bill* is
created once products have been ordered or received. For more information on managing vendor bills,
refer to :doc:`this documentation <manage>`.

.. note::
   After an order is placed, clicking :guilabel:`Receive Products` records the reception of new
   products into the database.

.. note::

>    With the **Inventory** app installed, confirming a *PO* automatically creates a receipt document,

Reviewer 2: ```suggestion
   With the **Inventory** app installed, confirming a |PO| automatically creates a receipt document,
```

======================
Requests for quotation
======================

> .. _inventory/purchase/manage_deals/request_for_quotation:

Reviewer 3: Small change to this custom anchor, for convention's sake–

- Update the app/module to Purchase, not Inventory
- Can abbreviate to "RFQ" to make the anchor shorter

```suggestion
.. _purchase/manage_deals/rfq:
```

.. |PO| replace:: :abbr:`PO (Purchase Order)`
.. |POs| replace:: :abbr:`POs (Purchase Orders)`
.. |RFQ| replace:: :abbr:`RFQ (Request for Quotation)`
.. |RFQs| replace:: :abbr:`RFQs (Requests for Quotation)`

> Odoo’s requests for quotation (RFQs) feature in the **Purchase** app standardizes
> ordering products from multiple vendors with varying prices and delivery times.

Reviewer 3: Remove curly quote (`’`), these pesky quotes can be carried over from copy/pasting from a Google doc or other formatted text editor.

Also there is an early line break in this sentence, can break the line closer to 100 chars :)
```suggestion
Odoo's requests for quotation (RFQs) feature in the **Purchase** app standardizes ordering products
from multiple vendors with varying prices and delivery times.
```

|RFQs| are documents companies send to vendors requesting product pricing. In Odoo, once the vendor
approves the |RFQ|, the purchase order (PO) is confirmed to align on lead times and pricing.

Configuration
=============

Product
-------

To auto-populate product information and prices on an |RFQ|, configure products by going to
:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a
new one by selecting :guilabel:`New`. Doing so opens the product form, where various sales and
purchasing data can be configured.

To configure purchasable products, tick the :guilabel:`Can be purchased` checkbox, under the product

> name. Then, go to the the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.

Reviewer 3: Remove double "the" :)
```suggestion
name. Then, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.
```

information auto-populate on an |RFQ| each time the product is listed.

Default columns include :guilabel:`Quantity`, :guilabel:`Price`, and :guilabel:`Delivery Lead Time`,
but other columns like, :guilabel:`Product Variant` or :guilabel:`Discounts`, can also be enabled.

To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon on
the right side of the header row to reveal a drop-down menu of additional columns that can be added
(or removed) from the :guilabel:`Purchase` tab.

.. note::
   Alternatively, prices and delivery lead times for existing products can be added in bulk by
   going to :menuselection:`Purchase app --> Configuration --> Vendor Pricelists`. Click
   :guilabel:`New` in the top-left corner. In the :guilabel:`Vendor` section of the pricelist form
   that appears, add the product information as it pertains to the vendor.

Order products
==============

With products and prices configured, follow these steps to create and send |RFQs| to make purchases
for the company.

|RFQ| dashboard
---------------

To get started, navigate to :menuselection:`Purchase app --> Orders --> Requests for Quotation`.

> The :guilabel:`Requests for Quotation` dashboard displays an overview of the company’s |RFQs|,

Reviewer 3: Remove smart quote
```suggestion
The :guilabel:`Requests for Quotation` dashboard displays an overview of the company's |RFQs|,
```

|POs|, and their status. The top of the screen breaks down all |RFQs| in the company, as well as
individual ones (where the user is the buyer) with a summary of their status.

> The top-right corner also provides a quick report of the company’s recent purchases by total value,

Reviewer 3: Remove smart quote
```suggestion
The top-right corner also provides a quick report of the company's recent purchases by total value,
```

lead times, and number of |RFQs| sent.

Additionally, the dashboard includes buttons for:

> :guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to vendor.

Reviewer 2: ```suggestion
:guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to the vendor.
```

:guilabel:`Waiting`: |RFQs| that have been sent by email, and are waiting on vendor confirmation.
:guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.

> .. image:: rfq/RFQ-dashboard.png

Reviewer 3: Please use all lowercase in filenames :)
```suggestion
.. image:: rfq/rfq-dashboard.png
```

   :align: center

>    :alt: |RFQ| dashboard with orders and order statuses.

Reviewer 3: Please no formatting (shorthand, italic, bold, guilabel, etc) in alt tags :)
```suggestion
   :alt: RFQ dashboard with orders and order statuses.
```

In addition to various view options, the :guilabel:`Requests for Quotation` dashboard provides
:guilabel:`Filters` and :guilabel:`Group By` options, accessible via the search bar drop-down menu.

> 

Reviewer 3: Consider adding a seealso to this documentation! :)
```suggestion
.. seealso::
   :doc:`../../../essentials/search`

```

Create new |RFQ|
----------------

To create a new |RFQ|, click the :guilabel:`New` button on the top-left corner of the
:guilabel:`Requests for Quotation` dashboard to reveal a new |PO| form.

Start by assigning a :guilabel:`Vendor`.

The :guilabel:`Vendor Reference` field points to the sales and delivery order numbers sent by the
vendor. This comes in handy once products are received, and the |PO| needs to be matched to the
delivery order.

The :guilabel:`Blanket Order` field refers to long-term purchase agreements on recurring orders with
set pricing. To view and configure blanket orders, head to :menuselection:`Purchase app --> Orders
--> Purchase agreements`.

The :guilabel:`Currency` can be changed, if purchasing products from a vendor in another country.

Next, configure an :guilabel:`Order Deadline`, which is the date by which the vendor must confirm
their agreement to supply the products.

.. note::
   After the :guilabel:`Order Deadline` is exceeded, the |RFQ| is marked as late, but the products
   can still be ordered.

:guilabel:`Expected Arrival` is automatically calculated based on the :guilabel:`Order Deadline`

> and vendor lead time. Click the checkbox for :guilabel:`Ask confirmation` to ask for signage at

Reviewer 2: ```suggestion
and vendor lead time. Tick the checkbox for :guilabel:`Ask confirmation` to ask for signage at
```

delivery.

With the :doc:`Storage Locations feature
<../../inventory/warehouses_storage/inventory_management/use_locations>` activated,
the :guilabel:`Deliver to` field appears, with options for the order shipment.

Select the receiving warehouse address here, or select :guilabel:`Dropship` to indicate that this
order is to be shipped directly to the end customer. When :guilabel:`Dropship` is selected, the
:guilabel:`Dropship address` field is enabled. Contact names auto-populate here from the
**Contacts** app.

Products tab
~~~~~~~~~~~~

> In the :guilabel:`Products`` tab, add the products to be ordered. Click :guilabel:`Add a product`,

Reviewer 2: ```suggestion
In the :guilabel:`Products` tab, add the products to be ordered. Click :guilabel:`Add a product`,
```

and type in the product name, or select the item from the drop-down menu.

To create a new product and add it, type the new product name in the :guilabel:`Product` column,

> select the :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the

Reviewer 2: ```suggestion
select :guilabel:`Create [product name]` from the resulting drop-down menu, and manually add the
```

unit price. Or, select :guilabel:`Create and edit...` to be taken to the product form for that new
item.

:guilabel:`Catalog` can also be selected to navigate to a product menu from the chosen vendor. From
here, products can be added to the cart.

> .. note::
>   To make adjustments to products and prices, access the product form by clicking the
>   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
>   the :guilabel:`Product` name.

Reviewer 3: Text in an admonition needs to align with the first letter of the admonitions name, `n` in `.. note::` in this case :)
```suggestion
.. note::
   To make adjustments to products and prices, access the product form by clicking the
   :icon:`oi-arrow-right` :guilabel:`(right arrow)` icon that becomes available upon hovering over
   the :guilabel:`Product` name.
```

Send |RFQ|
----------

Clicking :guilabel:`Send by Email` reveals a :guilabel:`Compose Email` pop-up window, with a
:guilabel:`Purchase: Request for Quotation` template loaded, ready to send to the vendor's email
address (configured in the **Contacts** app).

After crafting the desired message, click :guilabel:`Send`. Once sent, the |RFQ| moves to the
:guilabel:`RFQ Sent` stage.

Clicking :guilabel:`Print RFQ` downloads a PDF of the |RFQ|.

Confirm order
-------------

Clicking :guilabel:`Confirm Order` directly transforms the |RFQ| into an active |PO|.

.. tip::
   Odoo tracks communications on each order through the chatter of the |PO| form. This shows the
   emails sent between the user and the contact, as well as any internal notes and activities.

>    Messages, notes, and activities can also be logged on chatter.

Reviewer 2: ```suggestion
   Messages, notes, and activities can also be logged on the chatter.
```

Once an |RFQ| is confirmed, it creates a |PO|.

> On the new |PO|, the :guilabel:`Order Deadline` field changes :guilabel:`Confirmation Date`, which

Reviewer 2: ```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`, which
```

> On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`, which
> displays the date and time the user confirmed the order.

Reviewer 3: Need to break this line a little earlier, as "which" ends up being over 100 chars.
```suggestion
On the new |PO|, the :guilabel:`Order Deadline` field changes to :guilabel:`Confirmation Date`,
which displays the date and time the user confirmed the order.
```

Depending on the user's chosen configuration in the **Purchase** app settings, a *vendor bill* is
created once products have been ordered or received. For more information on managing vendor bills,

> refer to :doc:`this documentation <manage>`.

Reviewer 2: It could be beneficial to the reader to use the actual title of the Documentation that's being referenced here, just so they can also locate it on their own, if needed. This is just a suggestion, though. Not a requirement. Just something to consider during your revisions. 👍

approves the |RFQ|, the purchase order (PO) is confirmed to align on lead times and pricing.

Configuration
=============

Product
-------

To auto-populate product information and prices on an |RFQ|, configure products by going to
:menuselection:`Purchase app --> Products --> Products`. Select an existing product, or create a
new one by selecting :guilabel:`New`. Doing so opens the product form, where various sales and
purchasing data can be configured.

To configure purchasable products, tick the :guilabel:`Can be purchased` checkbox, under the product
name. Then, go to the :guilabel:`Inventory` tab, and enable the :guilabel:`Buy` route.

.. image:: rfq/product-vendor-pricelist-config.png
   :align: center
   :alt: Required configuration for purchasable products.

Vendor pricelist
----------------

In the :guilabel:`Purchase` tab of the product form, input the vendor and their price, to have this
information auto-populate on an |RFQ| each time the product is listed.

Default columns include :guilabel:`Quantity`, :guilabel:`Price`, and :guilabel:`Delivery Lead Time`,
but other columns like, :guilabel:`Product Variant` or :guilabel:`Discounts`, can also be enabled.

> To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(settings)` icon on
> the right side of the header row to reveal a drop-down menu of additional columns that can be added
> (or removed) from the :guilabel:`Purchase` tab.

Reviewer 3: Small change to update the icons label name, this icon is usually referred to as "additional options" :)
```suggestion
To enable or disable columns, click the :icon:`oi-settings-adjust` :guilabel:`(additional options)`
icon on the right side of the header row to reveal a drop-down menu of additional columns that can
be added (or removed) from the :guilabel:`Purchase` tab.
```

.. note::
   Alternatively, prices and delivery lead times for existing products can be added in bulk by
   going to :menuselection:`Purchase app --> Configuration --> Vendor Pricelists`. Click
   :guilabel:`New` in the top-left corner. In the :guilabel:`Vendor` section of the pricelist form
   that appears, add the product information as it pertains to the vendor.

Order products
==============

With products and prices configured, follow these steps to create and send |RFQs| to make purchases
for the company.

|RFQ| dashboard
---------------

To get started, navigate to :menuselection:`Purchase app --> Orders --> Requests for Quotation`.

The :guilabel:`Requests for Quotation` dashboard displays an overview of the company's |RFQs|,
|POs|, and their status. The top of the screen breaks down all |RFQs| in the company, as well as
individual ones (where the user is the buyer) with a summary of their status.

The top-right corner also provides a quick report of the company's recent purchases by total value,
lead times, and number of |RFQs| sent.

Additionally, the dashboard includes buttons for:

> :guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to the vendor.
> 
> :guilabel:`Waiting`: |RFQs| that have been sent by email, and are waiting on vendor confirmation.
> 
> :guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.

Reviewer 3: This seems like it should be formatted as a list, feel free to ignore!
```suggestion
- :guilabel:`To Send`: orders in the |RFQ| stage that have not been sent to the vendor.
- :guilabel:`Waiting`: |RFQs| that have been sent by email, and are waiting on vendor confirmation.
- :guilabel:`Late`: |RFQs| or |POs| where the :guilabel:`Order Deadline` has passed.
```
